### ISSUE TITLE
Questions about pre-trained models and prediction results in the paper

### ISSUE THREAD

**Issue by swb19** [StanfordASL/Trajectron-plus-plus#64]
Hello, your work is very helpful to me. However, I have some questions about the prediction accuracy achieved in the paper, when I used the pre-trained model you provided for evaluation, I found that its prediction accuracy is lower than that mentioned in Table 4 in the paper.
Could you please provide relevant information about your training parameters or pre-trained model at that time?

**Comment by BorisIvanovic** [comment]
Hi @swb19, thanks for your question! Have you seen [this issue](https://github.com/StanfordASL/Trajectron-plus-plus/issues/53#issue-1055666979)? Those results were achieved by someone else in the community running this codebase, and are more accurate to compare to with regards to the code in the current master branch.
